app/Models/Traits/HasCursor.php

```diff
@@ -55,10 +55,11 @@ public function scopeWhereAfterCursor($query, $cursor, $defaultSort = 'id,desc')
                     ->orWhere(function ($query) use ($column, $sortCursor, $id) {
                         // If the sorted cursor is null, we need to use a `WHERE NULL`
                         // query, since MySQL's `WHERE =` won't compare with `NULL`.
-                        $query->where(function ($query) use ($column, $sortCursor) {
-                            $query->where($column, '=', $sortCursor)
-                                ->orWhereNull($column);
-                        });
+                        if (is_null($sortCursor)) {
+                            $query->whereNull($column);
+                        } else {
+                            $query->where($column, '=', $sortCursor);
+                        }
```

Comment:
Oh ok right –– the reason I had changed this was because this doesn't handle cases where a page would overlap between sorted values (e.g. a page with 3, 2, 1, 1, 1, `null`, `null`, `null`). Without the `orWhereNull`, this page would not include those three nulls that it's supposed to.